views/pages/index.ejs

```diff
@@ -70,9 +75,9 @@
           <ul class="forecast-secondary">
             <% for (var i = 1; i < 12; i++) { %>
             <li>
-              <span class="data-secondary"><%= moment.unix(hourly.data[i].time).format('hA')%></span>
-              <span class="m-x-1"><%= Math.round(hourly.data[i].temperature) %>º</span>
-              <span><%- include('../partials/summary', {data: hourly.data[i]}) %></span>
+              <span class="data-secondary"><%= moment.unix(hourly.data[i + (dataIndex * 24)].time).format('hA')%></span>
+              <span class="m-x-1"><%= Math.round(hourly.data[i + (dataIndex * 24)].temperature) %>º</span>
```

Comment:
If this is 'tomorrow' (data index 1), and summary hourly list is 8, won't it be for tomorrow's 8pm weather instead?